src/web/browser/App.tsx

```diff
@@ -148,7 +154,7 @@ const App = ({ CAPI, NAV }: Props) => {
                 ))}
 
             <Portal root="cmp">
-                <CMP cmpUi={CAPI.config.switches.cmpUi} />
+                <CMP cmpUi={CAPI.config.cmpUi} />
```

Comment:
We might need to check if there is a semantic purpose to the switches property

Reply:
switches appear just to be boolean flags, so  I am not sure why they need thier own sub section in configs